plugins/wasi_nn/MLX/mlx/transformer.cpp

```diff
@@ -0,0 +1,14 @@
+#include "transformer.h"
+#include <mlx/ops.h>
+
+namespace WasmEdge::Host::WASINN::MLX {
+namespace mlx::core::nn {
+mx::array MultiHeadAttention::createAdditiveCausalMask(int N, mx::Dtype DType) {
+  auto Indices = mx::arange(N);
+  // mask = indices[:, None] < indices[None]
+  auto Mask = reshape(Indices, {N, 1}) < reshape(Indices, {1, N});
+  Mask = astype(Mask, DType) * -1e9;
```

Comment:
What does `-1e9` mean?

Reply:
It is usually set INF for the mask tensor. MLX implementation uses -1e9 instead since it is as good as INF and softmax(full(1e9)) != nan.

Reply:
Let's have some comments for this.